src/Popover.tsx

```diff
@@ -35,6 +36,7 @@ const Popover = React.forwardRef<View, PopoverProps>(function Popover(
     forceInitialAnimation = false,
     numberOfLines,
     visible = true,
+    caretStyle = null,
```

Comment:
No need to set a default value, `undefined` will work too:
```suggestion
    caretStyle,
```